Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/MediaControl.swift

```diff
@@ -218,13 +218,24 @@ open class MediaControl: UICorePlugin, UIGestureRecognizerDelegate {
         self.view.bindFrameToSuperviewBounds()
     }
 
-    func renderPlugins(_ plugins: [UICorePlugin]) {
-        plugins
-            .filter { $0 is MediaControlPlugin }
-            .map { $0 as! MediaControlPlugin }
-            .forEach { plugin in
+    func renderPlugins(_ plugins: [MediaControlPlugin]) {
+        if let mediaControlPluginOrderOption = core?.options[kMediaControlPluginsOrder] as? String {
+            let pluginsOrder = mediaControlPluginOrderOption
+                .replacingOccurrences(of: " ", with: "")
+                .components(separatedBy: ",")
+
+            var orderedPlugins: Array<MediaControlPlugin> = plugins.filter { pluginsOrder.contains($0.pluginName)}
+            orderedPlugins.append(contentsOf: plugins.filter { !pluginsOrder.contains($0.pluginName)})
+
+            orderedPlugins.forEach { plugin in
+                mediaControlView.addSubview(plugin.view, panel: plugin.panel, position: plugin.position)
+                plugin.render()
+            }
+        } else {
+            plugins.forEach { plugin in
```

Comment:
In my opinion, we could try to come up with a solution that ends up with a single:

```swift
plugins.forEach { plugin in
    mediaControlView.addSubview(plugin.view, panel: plugin.panel, position: plugin.position)
    plugin.render()
}
```

We could have this first step of checking the plugins order property, do all the sorting operation and then call the addSubview with whichever array of plugins we have.

My two cents: have another function called `sortPluginsIfNeeded` that returns the plugins in the right order, and then add them up to the interface:

```swift
let sortedPlugins = sortPluginsIfNeeded(plugins)
sortedPlugins.forEach { plugin in
    mediaControlView.addSubview(plugin.view, panel: plugin.panel, position: plugin.position)
    plugin.render()
}
```

or even pass the already sorted plugins to this `renderPlugins` function. 
Too many options, sorry! :joy:

Reply:
...or we can move to a Array of `MediaControlPlugin` extension. :)

Reply:
Done